harness/determined/cli/trial.py

```diff
@@ -138,33 +138,43 @@ def output_logs(args: Namespace):
     bundle = None
     if args.output_dir is None:
         output_dir = os.getcwd()
-        formatted_dt = datetime.now().strftime('%Y-%m-%d_%H:%M:%S')
-        tar = f"det-bundle-trail_{args.trial_id}-{formatted_dt}.tar.gz"
-        fullpath = os.path.join(output_dir, tar)
-        bundle = tarfile.open(fullpath, "w:gz")
 
     else:
         bundle = tarfile.open(f"{args.output_dir}.tar.gz", "w:gz")
 
+    filename_printf = "det-bundle-trial-{trial_id}-{datetime}.tar.gz"
+    tar_filename = filename_printf.format(
+        trial_id=args.trial_id, datetime=datetime.now().strftime("%Y%m%dT%H%M%S")
+    )
+    fullpath = os.path.join(output_dir, tar_filename)
+    bundle = tarfile.open(fullpath, "w:gz")
+
     bundle.add(trial_logs_filepath, arcname=os.path.basename(trial_logs_filepath))
     bundle.add(master_logs_filepath, arcname=os.path.basename(master_logs_filepath))
     bundle.add(api_trail_filepath, arcname=os.path.basename(api_trail_filepath))
     bundle.add(api_experiment_filepath, arcname=os.path.basename(api_experiment_filepath))
 
+    print(f"bundle directory: {bundle.name}")
```

Comment:
We are generating a file not a directory. Maybe just `print(f"bundle path: {fullpath}")`